improver/metadata/amend.py

```diff
@@ -61,14 +62,14 @@ def update_stage_v110_metadata(cube):
     cube.attributes["mosg__grid_version"] = "1.1.0"
 
 
-def amend_attributes(cube, attributes_dict):
+def amend_attributes(cube: Cube, attributes_dict: Dict[str, str]) -> None:
```

Comment:
```Dict[str, str]```the value portion of the dict is not necessarily a string.

Reply:
Fixed